app/views/course/experience_points_records/_experience_points_record.html.slim

```diff
@@ -2,3 +2,5 @@
   td = experience_points_record.points_awarded
   td = format_inline_text(experience_points_record.reason)
   td = link_to_user(experience_points_record.creator)
+  - if experience_points_record.manually_awarded? && can?(:destroy, experience_points_record)
```

Comment:
Do we want to have this check?

Should we just block all destroys of manually awarded experience points records instead?